packages/components/package.json

```diff
@@ -9,8 +9,9 @@
     ],
     "bin": {},
     "scripts": {
-        "prepack": "yarn clean && tsc",
+        "prepack": "yarn clean && tsc && yarn copy-files",
         "build": "yarn run prepack",
+        "copy-files": "copyfiles -u 1 src/assets/** dist/",
```

Comment:
There is no better way to do this? It seems a bit like a workaround more than a permanent solution

Reply:
There is no functionality in `tsc` to copy other files than .ts to the build directory. And `copyfiles` was the only working solution i could find.
There is an issue about this here https://github.com/Microsoft/TypeScript/issues/30835

Reply:
Thanks for the link, this seems to have worked and deployment is done!

> honestly, I have just spent hours and hours on a very weird problem just to realize that the reason was that copyfiles does not work the same way under linux and windows.
> Therefore, I would not recommend copyfiles especially if your build script is run as part of a docker build (under linux).
> === update =====
> Well, after further testing, it seems that adding double quotes should do the trick (hopefully):
> `"copy-files": "copyfiles -u 1 \"src/**/*.js\" lib"`